CMakeLists.txt

```diff
@@ -37,6 +40,23 @@ endif()
 # Dependencies
 # =============================================================================
 
+find_package(MPI)
+if (MPI_FOUND)
+    set(HDF5_PREFER_PARALLEL "ON")
+    add_definitions(-DHAVE_MPI)
+    include_directories (${MPI_INCLUDE_PATH})
+endif()
+
+find_package(HDF5)
+if (HDF5_FOUND)
+    if (HDF5_IS_PARALLEL)
+        include_directories (${HDF5_INCLUDE_DIRS})
+        set (HDF5_LIBS ${HDF5_LIBRARIES})
+    else()
+        message (STATUS "HDF5 found, but no parallel IO support was found, HDF5 support in replib will be disabled")
```

Comment:
```suggestion
        message (STATUS "HDF5 found, but no parallel IO support was found, HDF5 support in the reporting library will be disabled")
```

Reply:
'the reporting library' is an internal name that might not mean something to people using the library.

Reply:
Maybe "reports will be disabled"?